packages/mds-transaction-api/generate-flat-spec.ts

```diff
@@ -0,0 +1,15 @@
+import $RefParser from '@apidevtools/json-schema-ref-parser'
```

Comment:
I could modularize them a bit and toss them in `bin`, but for `generate-schemas.ts` there'll need to still be a file in the project dir which describes which schemas to grab for the filegen and imports them properly.

Reply:
Sure, but eventually I see us wanting to use this in other packages in which case not having to replicate the tooling is desirable. Parameterize them to accept a path or something like that.

Reply:
For sure, I'll see what I can do!

Reply:
Check out abadf93, I think that ought to be sufficient. Instead of dumping the scripts in `bin`, I put them into the schema validators package so they could be imported a bit more cleanly.